lib/active_model/serializer/adapter/json_api/link.rb

```diff
@@ -0,0 +1,34 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        class Link
```

Comment:
should other pieces of the JSON API spec have their own classes as well? (not that it should be part of this PR, but just wondering what the overall plan is)